Tribler/Test/Core/Modules/RestApi/test_my_channel_endpoints.py

```diff
@@ -35,18 +41,20 @@ def create_my_channel(self, name, description):
         self.channel_db_handler.on_channel_from_dispersy('fakedispersyid', None, name, description)
         return self.channel_db_handler.getMyChannelId()
 
-    def create_fake_channel(self, channel_name, channel_description):
+    def create_fake_channel(self, name, description, mode=u'closed', rss_url=None):
```

Comment:
I want to have the same method signature as the method in Session that we are replacing.
The same goes for create_fake_channel_with_existing_name